### ISSUE TITLE
【EESS】New submissions for Fri, 22 Dec 23

### ISSUE THREAD

**Issue by Yukeaaa** [Yukeaaa/arxiv-daily#1018]
## Keyword: volume render
There is no result 
## Keyword: volumetric render
There is no result 
## Keyword: remote render
There is no result 
## Keyword: hybrid render
There is no result 
## Keyword: raycast
There is no result 
### Responsible Deep Learning for Software as a Medical Device
 - **Authors:** Pratik Shah, Jenna Lester, Jana G Deflino, Vinay Pai
 - **Subjects:** Image and Video Processing (eess.IV); Computers and Society (cs.CY)
 - **Arxiv link:** https://arxiv.org/abs/2312.13333
 - **Pdf link:** https://arxiv.org/pdf/2312.13333
 - **Abstract**
 Tools, models and statistical methods for signal processing and medical image analysis and training deep learning models to create research prototypes for eventual clinical applications are of special interest to the biomedical imaging community. But material and optical properties of biological tissues are complex and not easily captured by imaging devices. Added complexity can be introduced by datasets with underrepresentation of medical images from races and ethnicities for deep learning, and limited knowledge about the regulatory framework needed for commercialization and safety of emerging Artificial Intelligence (AI) and Machine Learning (ML) technologies for medical image analysis. This extended version of the workshop paper presented at the special session of the 2022 IEEE 19th International Symposium on Biomedical Imaging, describes strategy and opportunities by University of California professors engaged in machine learning (section I) and clinical research (section II), the Office of Science and Engineering Laboratories (OSEL) section III, and officials at the US FDA in Center for Devices & Radiological Health (CDRH) section IV. Performance evaluations of AI/ML models of skin (RGB), tissue biopsy (digital pathology), and lungs and kidneys (Magnetic Resonance, X-ray, Computed Tomography) medical images for regulatory evaluations and real-world deployment are discussed.
### SE(3)-Equivariant and Noise-Invariant 3D Motion Tracking in Medical  Images
 - **Authors:** Benjamin Billot, Daniel Moyer, Neel Dey, Malte Hoffmann, Esra Abaci Turk, Borjan Gagoski, Ellen Grant, Polina Golland
 - **Subjects:** Image and Video Processing (eess.IV); Computer Vision and Pattern Recognition (cs.CV)
 - **Arxiv link:** https://arxiv.org/abs/2312.13534
 - **Pdf link:** https://arxiv.org/pdf/2312.13534
 - **Abstract**
 Rigid motion tracking is paramount in many medical imaging applications where movements need to be detected, corrected, or accounted for. Modern strategies rely on convolutional neural networks (CNN) and pose this problem as rigid registration. Yet, CNNs do not exploit natural symmetries in this task, as they are equivariant to translations (their outputs shift with their inputs) but not to rotations. Here we propose EquiTrack, the first method that uses recent steerable SE(3)-equivariant CNNs (E-CNN) for motion tracking. While steerable E-CNNs can extract corresponding features across different poses, testing them on noisy medical images reveals that they do not have enough learning capacity to learn noise invariance. Thus, we introduce a hybrid architecture that pairs a denoiser with an E-CNN to decouple the processing of anatomically irrelevant intensity features from the extraction of equivariant spatial features. Rigid transforms are then estimated in closed-form. EquiTrack outperforms state-of-the-art learning and optimisation methods for motion tracking in adult brain MRI and fetal MRI time series. Our code is available at github.com/BBillot/equitrack.
## Keyword: medical visualization
There is no result 
## Keyword: interactive volume
There is no result 
## Keyword: rendering
There is no result 
## Keyword: cinematic rendering
There is no result 
## Keyword: volume data
There is no result 
## Keyword: remote visualization
There is no result 
## Keyword: direct volume rendering
There is no result 
## Keyword: mobile device
There is no result 
## Keyword: transfer function
There is no result 
## Keyword: retrieval
There is no result 
## Keyword: video retrieval
There is no result 
### Optical Integrated Sensing and Communication: Architectures, Potentials  and Challenges
 - **Authors:** Yunfeng Wen, Fang Yang, Jian Song, Zhu Han
 - **Subjects:** Signal Processing (eess.SP); Information Theory (cs.IT)
 - **Arxiv link:** https://arxiv.org/abs/2312.13640
 - **Pdf link:** https://arxiv.org/pdf/2312.13640
 - **Abstract**
 Integrated sensing and communication (ISAC) is viewed as a crucial component of future mobile networks and has gained much interest in both academia and industry. Similar to the emergence of radio-frequency (RF) ISAC, the integration of free space optical communication and optical sensing yields optical ISAC (O-ISAC), which is regarded as a powerful complement to its RF counterpart. In this article, we first introduce the generalized system structure of O-ISAC, and then elaborate on three advantages of O-ISAC, i.e., increasing communication rate, enhancing sensing precision, and reducing interference. Next, waveform design and resource allocation of O-ISAC are discussed based on pulse waveform, constant-modulus waveform, and multi-carrier waveform. Furthermore, we put forward future trends and challenges of O-ISAC, which are expected to provide some valuable directions for future research.
## Keyword: smartphone
There is no result 
## Keyword: medical volume data
There is no result 
## Keyword: webgpu
There is no result 
## Keyword: webgl
There is no result 
## Keyword: pre-rendering
There is no result 
## Keyword: prerendering
There is no result 
## Keyword: motion prediction
There is no result 
## Keyword: incremental learning
There is no result 
## Keyword: svm incremental
There is no result 
## Keyword: nerf
There is no result 
## Keyword: multiorgan
There is no result 
## Keyword: multi-organ
There is no result 
## Keyword: multi organ
There is no result 
### One-Shot Initial Orbit Determination in Low-Earth Orbit
 - **Authors:** Ricardo Ferreira, Marta Guimarães, Filipa Valdeira, Cláudia Soares
 - **Subjects:** Systems and Control (eess.SY); Instrumentation and Methods for Astrophysics (astro-ph.IM); Machine Learning (cs.LG); Optimization and Control (math.OC)
 - **Arxiv link:** https://arxiv.org/abs/2312.13318
 - **Pdf link:** https://arxiv.org/pdf/2312.13318
 - **Abstract**
 Due to the importance of satellites for society and the exponential increase in the number of objects in orbit, it is important to accurately determine the state (e.g., position and velocity) of these Resident Space Objects (RSOs) at any time and in a timely manner. State-of-the-art methodologies for initial orbit determination consist of Kalman-type filters that process sequential data over time and return the state and associated uncertainty of the object, as is the case of the Extended Kalman Filter (EKF). However, these methodologies are dependent on a good initial guess for the state vector and usually simplify the physical dynamical model, due to the difficulty of precisely modeling perturbative forces, such as atmospheric drag and solar radiation pressure. Other approaches do not require assumptions about the dynamical system, such as the trilateration method, and require simultaneous measurements, such as three measurements of range and range-rate for the particular case of trilateration. We consider the same setting of simultaneous measurements (one-shot), resorting to time delay and Doppler shift measurements. Based on recent advancements in the problem of moving target localization for sonar multistatic systems, we are able to formulate the problem of initial orbit determination as a Weighted Least Squares. With this approach, we are able to directly obtain the state of the object (position and velocity) and the associated covariance matrix from the Fisher's Information Matrix (FIM). We demonstrate that, for small noise, our estimator is able to attain the Cram\'er-Rao Lower Bound accuracy, i.e., the accuracy attained by the unbiased estimator with minimum variance. We also numerically demonstrate that our estimator is able to attain better accuracy on the state estimation than the trilateration method and returns a smaller uncertainty associated with the estimation.
### Sample-based nonlinear detectability for discrete-time systems
 - **Authors:** Isabelle Krauss, Victor G. Lopez, Matthias A. Müller
 - **Subjects:** Systems and Control (eess.SY)
 - **Arxiv link:** https://arxiv.org/abs/2312.13658
 - **Pdf link:** https://arxiv.org/pdf/2312.13658
 - **Abstract**
 This paper introduces two sample-based formulations of incremental input/output-to-state stability (i-IOSS), a suitable detectability notion for general nonlinear systems. In this work we consider the case of limited output information, i.e., measurements are only infrequently and/or irregularly available. The output-dependent term of the sample-based i-IOSS bound is properly modified to yield a characterization for detectability in presence of incomplete output sequences. We provide both a non-timediscounted and a time-discounted formulation of samplebased i-IOSS. Furthermore, conditions for an i-IOSS system to be also sample-based i-IOSS are given and the relation between the two formulations of sample-based i-IOSS is shown.
### Millimeter-wave Radio SLAM: End-to-End Processing Methods and  Experimental Validation
 - **Authors:** Elizaveta Rastorgueva-Foi, Ossi Kaltiokallio, Yu Ge, Matias Turunen, Jukka Talvitie, Bo Tan, Musa Furkan Keskin, Henk Wymeersch, Mikko Valkama
 - **Subjects:** Signal Processing (eess.SP)
 - **Arxiv link:** https://arxiv.org/abs/2312.13741
 - **Pdf link:** https://arxiv.org/pdf/2312.13741
 - **Abstract**
 In this article, we address the timely topic of cellular bistatic simultaneous localization and mapping (SLAM) with specific focus on complete processing solutions from raw I/Q samples to user equipment (UE) and landmark location information in millimeter-wave (mmWave) networks. Firstly, we propose a new multipath channel parameter estimation solution which operates directly with beam reference signal received power (BRSRP) measurements, alleviating the need to know the true antenna beampatterns or the underlying beamforming weights. Additionally, the method has built-in robustness against unavoidable antenna sidelobes. Secondly, we propose new snapshot SLAM algorithms that have increased robustness and identifiability compared to prior-art, in practical built environments with complex clutter and multi-bounce propagation scenarios. The performance of the proposed methods is assessed at the 60 GHz mmWave band, via both realistic ray-tracing evaluations as well as true experimental measurements, in an indoor environment. Wide set of offered results clearly demonstrate the improved performance, compared to the relevant prior-art, in terms of the channel parameter estimation as well as the end-to-end SLAM performance. Finally, the article provides the measured 60 GHz data openly available for the research community, facilitating results reproducibility as well as further algorithm development.
### Leveraging Visual Supervision for Array-based Active Speaker Detection  and Localization
 - **Authors:** Davide Berghi, Philip J. B. Jackson
 - **Subjects:** Audio and Speech Processing (eess.AS); Machine Learning (cs.LG); Sound (cs.SD); Image and Video Processing (eess.IV); Signal Processing (eess.SP)
 - **Arxiv link:** https://arxiv.org/abs/2312.14021
 - **Pdf link:** https://arxiv.org/pdf/2312.14021
 - **Abstract**
 Conventional audio-visual approaches for active speaker detection (ASD) typically rely on visually pre-extracted face tracks and the corresponding single-channel audio to find the speaker in a video. Therefore, they tend to fail every time the face of the speaker is not visible. We demonstrate that a simple audio convolutional recurrent neural network (CRNN) trained with spatial input features extracted from multichannel audio can perform simultaneous horizontal active speaker detection and localization (ASDL), independently of the visual modality. To address the time and cost of generating ground truth labels to train such a system, we propose a new self-supervised training pipeline that embraces a ``student-teacher'' learning approach. A conventional pre-trained active speaker detector is adopted as a ``teacher'' network to provide the position of the speakers as pseudo-labels. The multichannel audio ``student'' network is trained to generate the same results. At inference, the student network can generalize and locate also the occluded speakers that the teacher network is not able to detect visually, yielding considerable improvements in recall rate. Experiments on the TragicTalkers dataset show that an audio network trained with the proposed self-supervised learning approach can exceed the performance of the typical audio-visual methods and produce results competitive with the costly conventional supervised training. We demonstrate that improvements can be achieved when minimal manual supervision is introduced in the learning pipeline. Further gains may be sought with larger training sets and integrating vision with the multichannel audio system.
